### DISCUSSION TITLE
Recommended framework for backend?

### DISCUSSION THREAD

**Discussion by rootfebri** [Oyelowo/surreal-orm#75] [General]
Im new on rustlang and trying to RIIR of my web backend from that written in php using laravel api framework.
If you don't mind, could you give me a list of recommended rust web framework using this library?
Thank you, I really excited to use this library.

**Comment by Oyelowo** [comment]
@rootfebri  
I am just seeing this 

- Axum
- Poem
- Actix
- Warp
- Rocket

I recommend Axum since that is built by the tokio team around the tokio, hyper stack.
Poem is built by creator of async_graphql and more feature-rich.
Others are great too if you like the API
